mysql_dump_script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+ 
+#date du jour
+DATE=`date +%y_%m_%d`
+ 
+#liste des dossier
+LISTEBDD=$( echo 'show databases' | mysql --user=root --password=AHJnd028.5m! )
+ 
+#on boucle sur chaque dossier (for découpe automatiquement par l'espace)
+for SQL in $LISTEBDD
+ 
+do
+ 
+if [ $SQL == "piwik" ]; then
+ 
+#echo $SQL
+mysqldump --user=root --password=AHJnd028.5m! $SQL | gzip > /home/$SQL"_mysql-new_4"$DATE.sql.gz
```

Comment:
This script file should not have a password written in it, but should instead read a password from configuration file or command argument. Also the output path and the file name do not look correct